src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,11 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
 import java.util.List;
 
 public interface MachineProducer {
-    /**
-     * In your implementations - create 2-3 models, add them to the list and return
-     * @return - the list of models
-     */
-    List<Object> get();
+    static List<Machine> get() {
```

Comment:
```suggestion
    List<T> get() {
```
тут краще не використовувати статику, це викошує поліморфізм в коді